### ISSUE TITLE
Implement an argument that prints the build output

### ISSUE THREAD

**Issue by greensn0w** [phpbrew/phpbrew#512]
This is more an idea than an issue.
I think we should implement an argument like `--verbose-build` that prints the configure and make.
It's really annoying to copy/paste the tail command every time you installed a missing dep.

Let's discuss this first, I'm very excited to hear what you think about this :smile: 

If we decide to include an argument like this, I can take over the part of coding this feature :smiley:

**Comment by marcioAlmada** [comment]
have you tried `--stdout` option? `phpbrew install --stdout ...`

**Comment by marcioAlmada** [comment]
Please reopen the issue if it's not what you want or if you have a better idea :)

**Comment by greensn0w** [comment]
It's nice to have this option but there are a few issues with it:
1. It's not in `phpbrew help`
2. It's name is confusing because `--stdout` could simply mean a `2>&1` alias
3. It's not working:
```shell
greensn0w@Mac  ~ 
 ❯ phpbrew install --stdout 5.3.29 +default
===> phpbrew will now build 5.3.29
PHP 5.3 requires +intl, enabled by default.
===> Loading and resolving variants...
Checking distribution checksum...
Checksum matched: 9469e240cbe6ac865aeaec89b253dd30
===> Extracting /Users/greensn0w/.phpbrew/distfiles/php-5.3.29.tar.bz2 to /Users/greensn0w/.phpbrew/tmp/php-5.3.29
===> Removing /Users/greensn0w/.phpbrew/build/php-5.3.29
===> Moving /Users/greensn0w/.phpbrew/tmp/php-5.3.29 to /Users/greensn0w/.phpbrew/build/php-5.3.29
===> Configuring 5.3.29...


Use tail command to see what's going on:
   $ tail -F /Users/greensn0w/.phpbrew/build/php-5.3.29/build.log


tee: illegal option -- -
usage: tee [-ai] [file ...]
./configure: line 1866: echo: write error: Broken pipe
./configure: line 1895: echo: write error: Broken pipe
./configure: line 1925: echo: write error: Broken pipe
```

**Comment by marcioAlmada** [comment]
> 1\. It's not in phpbrew help

It is, try `phpbrew help install`

> 2\. It's name is confusing because --stdout could simply mean a 2>&1 alias

I'd accept a pull request with --verbose then, I can't remember why it's not called `--verbose`... @c9s?

> 3\. It's not working:

Works with bash here. Can you describe your environment?

**Comment by c9s** [comment]
`--stdout` was added by @shinnya, yes there is a verbose option, but it's not equal to `--stdout`, the `--stdout` option outputs the build messages (which is not from Logger), @shinnya could you comment?

`phpbrew --verbose install ...` or `phpbrew --debug install...` the two options are for Logger written in PHP.

Looks like the `tee` command is broken, we shall fix it.

**Comment by greensn0w** [comment]
@marcioAlmada 
> Works with bash here. Can you describe your environment?

```shell
Darwin Mac-mini.local 14.3.0 
Darwin Kernel Version 14.3.0:
    Mon Mar 23 11:59:05 PDT 2015; 
root:xnu-2782.20.48~5/RELEASE_X86_64 x86_64

zsh 5.0.7 (x86_64-apple-darwin14.0.0)
```

The output of `env` is at -> http://git.io/vktDW

**Comment by marcioAlmada** [comment]
Thank you. I'll take a look ^^

**Comment by c9s** [comment]
I have the same issue too:

```
tee: illegal option -- -
usage: tee [-ai] [file ...]
./configure: line 1866: echo: write error: Broken pipe
./configure: line 1895: echo: write error: Broken pipe
./configure: line 1925: echo: write error: Broken pipe
```

**Comment by marcioAlmada** [comment]
Fixed with #598
